spacemacs/packages.el

```diff
@@ -543,7 +543,16 @@ which require an initialization must be listed explicitly in the list.")
   (use-package elisp-slime-nav
     :defer t
     :init
-    (add-hook 'emacs-lisp-mode-hook 'elisp-slime-nav-mode)))
+    (progn
+      (add-hook 'emacs-lisp-mode-hook 'elisp-slime-nav-mode)
+      (evil-leader/set-key-for-mode 'emacs-lisp-mode
+        "mgg" 'elisp-slime-nav-find-elisp-thing-at-point
+        "mhh" 'elisp-slime-nav-describe-elisp-thing-at-point))
+    :config
+    (progn
+      ;; mapping similarities to Vim
+      (evil-define-key 'normal emacs-lisp-mode-map (kbd "K") 'elisp-slime-nav-describe-elisp-thing-at-point)
+      (evil-define-key 'normal emacs-lisp-mode-map (kbd "C-t") 'elisp-slime-nav-find-elisp-thing-at-point))))
```

Comment:
I don't want to introduce these Vim like key bindings because they won't be consistent with the other languages. `mgg` and `mhh` are the way to go.